spec/spec-helper.coffee

```diff
@@ -120,27 +120,6 @@ warnIfLeakingPathSubscriptions = ->
     console.error("WARNING: Leaking subscriptions for paths: " + watchedPaths.join(", "))
   pathwatcher.closeAllWatchers()
 
-ensureNoDeprecatedFunctionsCalled = ->
```

Comment:
This might be nice to keep around for when Atom 2.0 approaches and we start deprecating stuff again.